phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/apps/ViewApps.java

```diff
@@ -141,7 +137,8 @@ public void setOnRetryButtonClicked(EventHandler<? super MouseEvent> event) {
     private void showAppDetails(ApplicationDTO application, SettingsManager settingsManager) {
         final AppPanel appPanel = new AppPanel(application, themeManager, settingsManager);
         appPanel.setOnScriptInstall(this::installScript);
-        showRightView(appPanel);
+        appPanel.setMaxWidth(400);
```

Comment:
I just did it here because the app details were expanding too much. E.g. for the engines it was not necessary. It's not the smartest solution as it won't scale if you scale the UI but I think we can take care of that later on.